zulipterminal/ui_tools/views.py

```diff
@@ -1216,7 +1216,8 @@ def __init__(self, controller: Any, msg: Message, title: str,
         self.message_links = message_links
         self.time_mentions = time_mentions
         date_and_time = controller.model.formatted_local_time(
-                                        msg['timestamp'], show_year=True)
+            msg['timestamp'], show_seconds=True, show_year=True
```

Comment:
You might have observed, that you had to change the parameter you added, in both the commits, when you refactor in a later commit. See my review comment for reference :)

Reply:
Sorry, I didn't quite catch what you're trying to say. What do you mean by refactor in a later commit? I didn't have to make any changes in this file in the second commit.

Reply:
Ah, that is surely a confusion. See my comment [here](https://github.com/zulip/zulip-terminal/pull/930/commits/45fda745a561e50f8dd4d3d42c85e323d10e14eb#r579353017) Apologies :sweat_smile: